src/Middleware/Localization/src/RequestLocalizationMiddleware.cs

```diff
@@ -109,23 +110,13 @@ public async Task Invoke(HttpContext context)
                         continue;
                     }
 
-                    if (cultureInfo == null && uiCultureInfo != null)
-                    {
-                        cultureInfo = _options.DefaultRequestCulture.Culture;
-                    }
-                    else if (cultureInfo != null && uiCultureInfo == null)
-                    {
-                        uiCultureInfo = _options.DefaultRequestCulture.UICulture;
-                    }
+                    cultureInfo ??= _options.DefaultRequestCulture.Culture;
+                    uiCultureInfo ??= _options.DefaultRequestCulture.UICulture;
```

Comment:
💭 This seems to be a slight change in behavior, but I'm not sure if it's a problem or not.

Reply:
Line 107 asserts that the two cannot be simultaneously null. The null checks inside the if reiterate these.